config.go

```diff
@@ -75,5 +77,14 @@ func Parse(path string) (*Config, error) {
 		c.Godoc.Host = host
 	}
 
+	// set default branch
+	for v := range c.Packages {
+		p := c.Packages[v]
+		if p.Branch == "" {
```

Comment:
Minor: This can be simplified a bit.
```suggestion
	for v, p := range c.Packages {
		if p.Branch == "" {
```